src/main/java/narcibot/Event.java

```diff
@@ -0,0 +1,73 @@
+package narcibot;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class Event extends Task {
+    private String time;
+
+    /**
+     * Constructor for event class
+     * @param name name of the event
```

Comment:
Shouldn't there be a new line after the description of your javadocs? 🤔
I noticed this in several parts of your code.

Reply:
Shouldn't there be a new line separating your description and the parameters in your javadocs? 🤔 I noticed the same issue in several other places too.